### DISCUSSION TITLE
Prefix editing and Vlan editing UI not consistant

### DISCUSSION THREAD

**Discussion by patrick-GH** [netbox-community/netbox#9773] [General]
Hello, I just noticed while adding a new prefix and vlan to netbox, the UI for adding prefixes and vlans are different

In the prefixes section the Site/Vlan assignment is one column as shown below:
![image](https://user-images.githubusercontent.com/39420482/179623633-7df500f3-1ef0-4fda-a225-5c2861c4f396.png)

In the vlan section it's setup as a tab setup, but it isn't very noticeable that "site" is a button
![image](https://user-images.githubusercontent.com/39420482/179623917-e5ced79e-8a53-4efb-ac9f-30add043c48d.png)

If the editing UI could be consistent or if your going to go with the tab view, then have other tabs boxed such as you do with "cancel" or "create & add another"

I'm using an older version 3.0.12, so if this is addressed in new releases, that's great!

**Comment by DanSheps** [comment]
There is a reason for this:  VLANs can be assigned to either a VLAN group, or a site (but not both).  A Prefix on the other hand can be assigned to both a VLAN and a Site.

This is the reason for the different UI's
